src/runtime/pay-flow-test.js

```diff
@@ -235,7 +235,7 @@ describes.realWin('PayStartFlow', {}, env => {
     analyticsMock.expects('setSku').withExactArgs('newSku');
     eventManagerMock
       .expects('logSwgEvent')
-      .withExactArgs(AnalyticsEvent.ACTION_SUBSCRIBE, true, null);
+      .withExactArgs(AnalyticsEvent.ACTION_PAYMENT_FLOW_STARTED, true, null);
```

Comment:
could we drop the final , null for all these calls?  If we are just going to use the default anyway I feel like we might as well.